actions.md

```diff
@@ -0,0 +1,547 @@
+        "binary": false
+    },
+    "annotations": [
+        {
+            "key": "exec",
+            "value": "nodejs:6"
+        }
+    ],
+    "limits": {
+        "timeout": 60000,
+        "memory": 256,
+        "logs": 10
+    },
+    "publish": false
+}
+```
+
+### Get the URL for an action
+
+An action can be invoked through the REST interface via an HTTPS request.
+
+To get an action URL, execute the following command:
+
+```
+nim action get greeting --url
+```
+
+A URL with the following format will be returned for standard actions:
+```
+ok: got action actionName
+https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/greeting
+```
+
+**[[NH: check link]]**
+Authentication is required when invoking an action via an HTTPS request using this resource path. For more information regarding action invocations using the REST interface, see [Using REST APIs with OpenWhisk](rest_api.md#actions).
+
+**[[NH: check link:]]**
+Another way of invoking an action that does not require authentication is via [web actions](webactions.md#web-actions).
+
+Any action may be exposed as a web action, using the `--web true` command line option when creating or updating an action.
+
+```
+nim action update greeting --web true
+ok: updated action greeting
+```
+
+The resource URL for a web action is different:
+```
+nim action get greeting --url
+ok: got action greeting
+https://${APIHOST}/api/v1/web/${NAMESPACE}/${PACKAGE}/greeting
+```
+
+You can use `curl` or `wget` to invoke the action.
+```
+curl `nim action get greeting --url | tail -1`.json
+{
+  "payload": "Hello, Toto from somewhere!"
+}
+```
+
+### Save action code
+
+Code associated with an existing action may be retrieved and saved locally. Saving can be performed on all actions except sequences and docker actions.
+
+1. Save action code to a filename that corresponds with an existing action name in the current working directory. A file extension that corresponds to the action kind is used. An extension of _.zip_ is used for action code that is a zip file.
+
+  **[[NH: please check command and output:]]**
+  ```
+  nim action get /whisk-system/samples/greeting --save
+  ok: saved action code to /path/to/namespace/greeting.js
+  ```
+
+2. You can provide your own file name and extension by using the `--save-as` flag.
+
+  **[[NH: check output:]]**
+  ```
+  wsk action get /whisk-system/samples/greeting --save-as hello.js
+  ok: saved action code to /path/to/namespace/hello.js
+  ```
+
+### List actions
+
+You can list all the actions that you have created using the following commnand:
+
+```
+nim action list
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+Here, we see actions listed in order from most to least recently updated. For easier browsing, you can use the flag `--name-sort` or `-n` to sort the list alphabetically:
+
+**[[NH: Output list seems backwards for alphabetical order?]]**
```

Comment:
Output format has changed recently (don't worry about that ... I think it is not yet in the version of `nim` that you can download and install).  I will be rectifying that soon, but, I see no problem with `--name-sort` when I try it (on a different namespace).   Is this something you executed or is it copied from the Apache document?

Reply:
It came over from the Apache doc.